hello.py

```diff
@@ -12,6 +12,12 @@ def add(number1, number2):
 word = input()
 
 
+def check_letter(word):
+    word_counter = 0
+    for letter in word:
+        if letter == 'a':
```

Comment:
What about uppercase words? Does it work then too?

Reply:
Resolved that issue